zerver/tests/test_subs.py

```diff
@@ -3210,7 +3210,7 @@ def test_subscriptions_query_count(self) -> None:
                 dict(principals=ujson.dumps([user1.email, user2.email])),
                 invite_only=True,
             )
-        self.assert_length(queries, 40)
+        self.assert_length(queries, 52)
```

Comment:
These numbers going up by 12 means you have a bug where you've introduced database queries in a loop.